extensions/resource-deployment/src/interfaces.ts

```diff
@@ -284,9 +285,6 @@ export interface FieldInfo extends SubFieldInfo, FieldInfoBase {
 	defaultValue?: string;
 	confirmationRequired?: boolean;
 	confirmationLabel?: string;
-	textValidationRequired?: boolean;
```

Comment:
this is a breaking change, not that it will cause compilation errors, the places using this will be ignored, have you done a global search to find all the occurrences of these properties? for instance, I don't see: https://github.com/microsoft/azuredatastudio/blob/main/extensions/asde-deployment/package.json#L342 being updated here. also, please email the team and ask whether they have extensions not inside ADS repo leveraging this.

Reply:
Will do. Do you think it would be prudent to leave the implementation of old syntax also?

Reply:
right now there are only a few extensions using resource deployment features, I'd prefer to do the clean up now otherwise we will have to live with it forever.

Reply:
Made the change to ASDE.